src/Core/View.js

```diff
@@ -224,6 +229,9 @@ View.prototype.addLayer = function addLayer(layer, parentLayer) {
 View.prototype.notifyChange = function notifyChange(needsRedraw, changeSource) {
     this._changeSources.add(changeSource);
     this.mainLoop.scheduleViewUpdate(this, needsRedraw);
+    if (needsRedraw) {
+        this.dispatchEvent({ type: VIEW_EVENTS.UPDATED });
```

Comment:
Can you document the purpose of this event?
And/or give example use case for it?